.travis.yml

```diff
@@ -24,6 +24,20 @@ matrix:
       env: CMAKEFLAGS="-DWITH_BOOST=ON -DHAS_MEMRCHR=OFF" CXX=clang++
     - compiler: gcc
       os: linux
+      dist: precise
+      sudo: false
+      addons:
+        apt:
+          sources:
+            - george-edison55-precise-backports
+          packages:
+            - cmake
+            - cmake-data
```

Comment:
Because cmake-data isn't marked as installed automatically on Travis' infrastructure, thus trying to upgrade `cmake` fails if you don't explicitly try to upgrade `cmake-data` at the same time.